samples/js/js-customer-registry-quickstart/.gitignore

```diff
@@ -0,0 +1,5 @@
+/lib/generated
+/node_modules
+/user-function.desc
+# test quickstarts without lock file
+/package-lock.json
```

Comment:
Not committing the package lock file for this sample, so that testing is similar to user download.